examples/powertools-examples-core/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id("io.freefair.aspectj.post-compile-weaving") version "6.6.3"
+    kotlin("jvm") version "1.9.10"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("com.amazonaws:aws-lambda-java-core:1.2.2")
+    implementation("com.fasterxml.jackson.core:jackson-annotations:2.13.2")
+    implementation("com.fasterxml.jackson.core:jackson-databind:2.13.2.2")
+    implementation("com.amazonaws:aws-lambda-java-events:3.11.0")
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.2")
+    aspect("software.amazon.lambda:powertools-tracing:1.17.0")
```

Comment:
These should be set to the current snapshot version used in all the other POMs, so they build from local maven cache. The build of the example should fail if we make a breaking change to the local copy of the library itself I am working on.

This'll then also get automagically rolled forward as part of our release process. The link to the examples from the main page of the repo goes to a tagged version, which then has proper release artifact versions (e.g., 1.18.0 again, without snapshot), so the subdirs can still be copied and run locally.

Reply:
That is possible, but probably a bit tricky. Since this Gradle project is not connected to Maven project directly, we'll have to trigger Maven build explicitly to populate the cache with the snapshot of the library.

Reply:
Github actions: that's exactly how the Gradle build works at the moment
End users: should use the link from the README to find the samples - then they get the latest tagged version, wherein the build process has replaced all the SNAPSHOT references with release ones, and everything works - [check this out](https://github.com/aws-powertools/powertools-lambda-java/tree/v1.17.0/examples) - `powertools.version` [is set to](https://github.com/aws-powertools/powertools-lambda-java/blob/v1.17.0/examples/powertools-examples-core/sam/pom.xml#L6)